manifests/params.pp

```diff
@@ -1,15 +1,6 @@
 # Candlepin params
 class candlepin::params {
-  $tomcat = $::osfamily ? {
-    /^(RedHat|Linux)/ => $::operatingsystem ? {
-      'Fedora'  => 'tomcat',
-      default   => $::operatingsystemrelease ? {
-        /^7\./  => 'tomcat',
-        default => 'tomcat6'
-      }
-    },
-    default => 'tomcat'
-  }
+  $tomcat = 'tomcat'
```

Comment:
We could also update this in puppet-certs (https://github.com/Katello/puppet-certs/blob/efd24fe48015cf54e6b84a345d473add3408a337/manifests/params.pp#L4)